cognite_toolkit/config.yaml

```diff
@@ -128,6 +112,9 @@ cognite_modules:
     cdf_apm_base:
       apm_datamodel_space: APM_SourceData
       apm_datamodel_version: '1'
+  scheduleHourly: 7 * * * *
```

Comment:
I see the core module has been placed in-between the top level variables. Suggest, we sort the config to either have all modules first and then top level or first to level and then all modules. I lean towards having all the top level variables on top. Wdyt?